src/compiler/emitter.ts

```diff
@@ -3305,20 +3305,20 @@ namespace ts {
             return getSourceTextOfNodeFromSourceFile(currentSourceFile, node, includeTrivia);
         }
 
-        function getLiteralTextOfNode(node: LiteralLikeNode): string {
+        function getLiteralTextOfNode(node: LiteralLikeNode, neverAsciiEscape: boolean): string {

```

Comment:
Can you make neverAsciiEscape Boolean | undefined so that in above change you don't need to pass || false?